dms-infrastructure/src/main/kotlin/team/aliens/dms/global/security/SecurityConfig.kt

```diff
@@ -223,6 +223,14 @@ class SecurityConfig(
                     .requestMatchers(HttpMethod.GET, "/volunteers").hasAuthority(STUDENT.name)
                     .requestMatchers(HttpMethod.GET, "/volunteers/my/application").hasAuthority(STUDENT.name)
 
+                authorize
+                        // /vote
+                    .requestMatchers(HttpMethod.POST,"/votes").hasAnyAuthority(MANAGER.name)
+                    .requestMatchers(HttpMethod.GET,"/votes/{voting-topic-id}").hasAnyAuthority(MANAGER.name)
+                    .requestMatchers(HttpMethod.DELETE,"/votes/{voting-topic-id}").hasAnyAuthority(MANAGER.name)
+                    .requestMatchers(HttpMethod.PATCH,"/votes/{voting-topic-id}").hasAnyAuthority(MANAGER.name)
+                    .requestMatchers(HttpMethod.GET,"/votes").hasAnyAuthority(MANAGER.name,STUDENT.name)

```

Comment:
```suggestion
                    // /vote
                    .requestMatchers(HttpMethod.POST,"/votes").hasAnyAuthority(MANAGER.name)
                    .requestMatchers(HttpMethod.GET,"/votes/{voting-topic-id}").hasAnyAuthority(MANAGER.name)
                    .requestMatchers(HttpMethod.DELETE,"/votes/{voting-topic-id}").hasAnyAuthority(MANAGER.name)
                    .requestMatchers(HttpMethod.PATCH,"/votes/{voting-topic-id}").hasAnyAuthority(MANAGER.name)
                    .requestMatchers(HttpMethod.GET,"/votes").hasAnyAuthority(MANAGER.name,STUDENT.name)

```
권한이 하나일 경우에는 hasAuthority로 수정해주세요....

Reply:
수정했습니다.